build-certificates-dir.sh

```diff
@@ -23,7 +23,9 @@ if [[ $PKG_NAME = osg-ca-certs ]]; then
 fi
 
 ../mk-index.pl --version "$OUR_CERTS_VERSION" --dir "$CADIST" \
-                 --out "$CADIST/INDEX" -format 1 --style new
+               --out "$CADIST/INDEX" -format 1 --style new    \
+               --igtf_version "${IGTF_CERTS_VERSION}IGTFNEW"  \
+               --osg_version  "${OSG_CERTS_VERSION}NEW"
```

Comment:
Should this be `${OUR_CERTS_VERSION}` or is the comment on line 5 wrong?

Reply:
Ah, good point, the comment on line 5 was ... incomplete :)

But i do mean the `${OSG_CERTS_VERSION}` -- the `OUR` is the version for the current build (maybe igtf or maybe osg), but in order to get the links in the html right, the mk-index script needs to know about both.

Anyway i've updated the comment!